sdk/dotnet/Pulumi.Automation/Events/Events.cs

```diff
@@ -0,0 +1,495 @@
+// Copyright 2016-2021, Pulumi Corporation
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+
+// NOTE: The classes in this file are intended to align with the serialized
+// JSON types defined and versioned in sdk/go/common/apitype/events.go
+namespace Pulumi.Automation.Events
+{
+    // TODO(vipentti): Split to separate files?
+
```

Comment:
Should the Events be split into separate files per typical C# standards?

Reply:
I'll let C# people speak to this @orionstudt @t0yv0 @justinvp @mikhailshilkov - but in general, I would say yes, follow language standards where appropriate.